bundles/org.openhab.core.io.rest.sitemap/src/main/java/org/openhab/core/io/rest/sitemap/internal/WidgetLabelSource.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.io.rest.sitemap.internal;
+
+import org.openhab.core.model.sitemap.sitemap.Widget;
+
+/**
+ * @author Danny Baumann - Initial contribution
+ */
+public enum WidgetLabelSource {
+    SitemapDefinition,
+    ItemLabel,
+    None;
```

Comment:
The convention is rather to use uppercase.

Reply:
Fixed.